src/core/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ file(GLOB EspressoCore_SRC
 
 if( WITH_COVERAGE )
   set(CMAKE_BUILD_TYPE "Debug" CACHE STRING "Choose the type of build, options are: Debug Release RelWithDebInfo MinSizeRel." FORCE)
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -O0 --coverage -fprofile-arcs -ftest-coverage")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -O1 --coverage -fprofile-arcs -ftest-coverage")
```

Comment:
That is not a good idea.  `-O1` includes for example `-finline-functions-called-once` which will mess up coverage.

Reply:
From the numbers that does not seem to be a big issue.